storage/bootstrap.go

```diff
@@ -193,7 +206,7 @@ func (m *bootstrapManager) bootstrap() error {
 	// all the data we bootstrapped.
 	rateLimitOpts := m.fsManager.RateLimitOptions()
 	m.fsManager.SetRateLimitOptions(rateLimitOpts.SetLimitEnabled(false))
-	m.fsManager.Run(m.nowFn(), false)
+	m.fsManager.Run(m.nowFn(), false, true)
```

Comment:
Can this be done without needing`force`? This is the only place that uses it afaict, and fsManager will be stopped by virtue of the loop above.